integrationTests/vm/arwen/arwenVM_test.go

```diff
@@ -8,12 +8,18 @@ import (
 	"testing"
 	"time"
 
+	"github.com/ElrondNetwork/elrond-go/core"
 	"github.com/ElrondNetwork/elrond-go/data/transaction"
 	"github.com/ElrondNetwork/elrond-go/integrationTests/vm"
+	"github.com/ElrondNetwork/elrond-go/logger"
 	"github.com/ElrondNetwork/elrond-go/process/factory"
 	"github.com/stretchr/testify/assert"
 )
 
+func init() {
+	_ = logger.SetLogLevel("*:INFO,process/smartcontract:DEBUG")
```

Comment:
this is fine for a debugging session, might remove it as it pollutes with text when running the tests

Reply:
yes. and it slows down the benchmarking.